tests/unit/stack_stores/test_stack_stores.py

```diff
@@ -126,7 +126,7 @@ def test_register_deregister_components(
     orchestrator = stack_store.get_stack_component(
         StackComponentType.ORCHESTRATOR, "local_orchestrator"
     )
-    assert orchestrator.flavor == OrchestratorFlavor.LOCAL
+    assert orchestrator.flavor == "local"
```

Comment:
Was there any reasoning of why we took out the enum here? Raising it in case it was unintentional

Reply:
Yes. All of the different types of flavors have been removed for the purpose of extensibility. In more detail, previously, when you wanted to implement a new stack component as a user, you had to come up with a flavor that would be included within the corresponding enum (some parts of the code were dependent on the fact that this variable was an enum, not str). This forced our users to come up with a way to extend our built-in enums otherwise there were some inconsistencies. Now, they can simply use a string as the enum doesn't really provide any additional functionalities.